app/db/models/user.js

```diff
@@ -60,6 +60,14 @@ module.exports = (sequelize, DataTypes) => {
         {
           unique: true,
           fields: ['email']
+        },
+        {
+          unique: true,
+          fields: ['auth0_id']
```

Comment:
not 100% sure if these fields actually are unique without checking the db in heroku, which is down atm, but I think they should be

Reply:
Heroku should be back up, any chance you could double check this? Thanks!